cointoss/db/migrate/20200808133128_associate_bets_with_user.rb

```diff
@@ -0,0 +1,7 @@
+class AssociateBetsWithUser < ActiveRecord::Migration[5.2]
+  def change
+    change_table :bets do |t|
+      t.belongs_to :user, foreign_key: true
```

Comment:
make sure bets are associated with a user and room

Reply:
Yeah, I think the Bet model is missing `belongs_to :user`